src/test/regress/expected/multi_insert_select.out

```diff
@@ -2392,6 +2392,39 @@ SELECT * FROM ref_table ORDER BY user_id, value_1;
 (5 rows)
 
 DROP TABLE ref_table;
+-- Select from reference table into reference table
+CREATE TABLE ref1 (d timestamptz);
+SELECT create_reference_table('ref1');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+CREATE TABLE ref2 (d date);
+SELECT create_reference_table('ref2');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+INSERT INTO ref2 VALUES ('2017-10-31');
+INSERT INTO ref1 SELECT * FROM ref2;
+SELECT count(*) from ref1;
+ count 
+-------
+     1
+(1 row)
+
+-- also test with now()
+INSERT INTO ref1 SELECT now();
```

Comment:
this probably goes through INSERT...SELECT on the coordinator since it is not selecting from a distributed or reference table.

Reply:
hmm, that's right. Should I just drop this now() test or add as a second column?

Reply:
something like `SELECT now() FROM ref2` should work

Reply:
Done